client/modules/food/reducers/packing.js

```diff
@@ -44,6 +48,19 @@ export const pack = newPackedCustomers => dispatch => {
     .catch(error => dispatch(packFailure(error)))
 }
 
+export const deliveredPackage = singlePackage => dispatch => {
```

Comment:
This can just be:
```js
export const deliveredPackage = packageId => ({
  [CALL_API]: {
    endpoint: 'packing',
    method: 'PUT',
    body: {packageId},
    responseSchema: packageSchema,
    types: [DELIVERED_REQUEST, DELIVERED_SUCCESS, DELIVERED_FAILURE]
  }
})
```

The api returns a package, it normalizes it with the package schema and updates the package entities in the store, so the ui updates.

Reply:
@jspaine I made the updates, but it seems that I need to change the responseSchema to arrayOfPackages for the UI to update. Please confirm if you agree and I'll wrap up the updates.

Reply:
Ah, it should work if you respond with single package from the controller, are you sure you changed the response in the controller to `res.json(deliveredPackage)` rather than `res.json({packages: deliveredPackage})`?

Otherwise just push it and I'll take a look. Thanks!